### PR TITLE
Teddy with hidden storage

### PR DESCRIPTION
**Pull request by Autisem**
## About The Pull Request
request of Mateja#2274, this teddy bear can be cut open to store things into, and then sown back up with the new needle
TODO: propper sprites, its only debug stuff for now

## Why It's Good For The Game
A neat way to hide your emags I suppose?

## Changelog
:cl:
add: Teddy bear with hidden storage
add: needle and thread
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/storage/internal.dm [hunk 1]
```diff
@@ -72,3 +72,69 @@
...
+			to_chat(user,"You cut open the teddy bear")
+			icon_state = icon_open
+			open = TRUE
+			update_icon()
+			return FALSE
+		else
+			return FALSE
+	if(open && istype(W, key))
+		open = FALSE
+		to_chat(user,"You sow the teddy shut")
```
Comment (MadmanMartian) [inline code/game/objects/items/weapons/storage/internal.dm:126]: needs a SPAN_NOTICE() around the string, and it's sew not sow

File: code/game/objects/items/weapons/storage/internal.dm [hunk 2]
```diff
@@ -72,3 +72,69 @@
...
+/obj/item/weapon/teddy/Destroy()
+	qdel(container)
+	container = null
+	. = ..()
+
+/obj/item/weapon/teddy/attack_hand(mob/user as mob)
+	if(open)
+		if (is_held() && !container.handle_attack_hand(user))
+			return TRUE
+	..(user)
```
Comment (MadmanMartian) [inline code/game/objects/items/weapons/storage/internal.dm:107]: Function args are already passed implicitely

**Comment by Kurgis** [comment]
More of a question to passing maintainers, when are we going to make storage a component for similar items in the future?

**Comment by drexample** [comment]
@Autisem resolve conflict

**Comment by TorinoFermic** [comment]
Please resolve the conflict.

**Comment by SpadesSlick** [comment]
Will this PR be fixed?

**Comment by drexample** [comment]
@Autisem this PR will be closed in 7 days.

**Pull request closed without merging**